docs/UserGuide.md

```diff
@@ -212,7 +212,7 @@ Format: `delete INDEX`
 * There is a confirmation popout after the user enters the command but before the delete is actually done.
   click `OK` to continue or `Cancel` to abort the deletion.
 
-![delete popout](images/deleteConfirmation.png)
+![delete popup](images/Delete popup.png)
```

Comment:
Would it be better to add the image as <img src="images/Deletepopup.png" alt="Delete popup screenshot" height=auto width="850"/> to standardise the screenshot size, changing the "Delete popup.png" filename to "Deletepopup.png"?